app/templates/courses/classroom-settings-modal.pug

```diff
@@ -157,12 +157,17 @@ block modal-body-content
               i.spl.text-muted(data-i18n="signup.optional")
           select.form-control(name="type", value=view.classroom.get('type'))
             option(value="" data-i18n="courses.avg_student_exp_select")
-            option(value="in-school" data-i18n="courses.class_type_in_school")
+            if me.isCodeNinja()
+              option(value="camp-esports" data-i18n="courses.class_type_camp_esports")
```

Comment:
current is fine. also i think  we can extend this type to be used overall so that esports classroom can get rid of some other classroom features and behaviors more like a student-oriented clan